schema/deploy/tables/revision_status.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy cif:tables/revision_status to pg
```

Comment:
You might want to add the sqitch dependency on `tables/amendment_status`, for future devs to know where this is coming from